src/av2/structures/ndgrid.py

```diff
@@ -142,16 +145,23 @@ def points_to_bev_img(
 
         points_xy = points[..., :2].copy()  # Prevent modifying input.
         indices = self.transform_to_grid_coordinates(points_xy)
-        indices_int, _ = crop_points(indices, lower_bound_inclusive=(0.0, 0.0), upper_bound_exclusive=self.dims)
+        indices_int, valid_mask = crop_points(
```

Comment:
I think that the indices become integer coordinates in the line prior, right? We might want to move that new variable name up a line to signify that.

Reply:
Maybe a better renaming would be `indices_int` to `indices_crop` here.